scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -186,6 +196,19 @@ class FossId(
 
         log.info { "waitForResult parameter is set to '$waitForResult'" }
 
+        val namingProjectPattern = fossIdScannerOptions["namingProjectPattern"]?.apply {
```

Comment:
Nit: Using `let` (and `it` in the next line) would be a bit more common for this use-case, IMO. Same below.

Reply:
Understood but then you have to return something from `let` i.e.

```
val namingProjectPattern = fossIdScannerOptions["namingProjectPattern"]?.let {
            FossId.log.info { "Naming pattern for projects is $it." }
            it
}
```
which is more code or am I missing something ?

Reply:
My bad, you're right. I missed that.